### PR TITLE
Updates to original PFA Password Management App

### PR DESCRIPTION
**Pull request by thedrewery**

### CODE REVIEW

**Review by hanswand — APPROVED** [review]
great implementation

File: __tests__/CompTests/passwordEntryTest.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+});
+
+test('calls correct function on click', () => {
+  const onClick = jest.fn();
+  const { getByText } = render(<PasswordEntry {...defaultProps} onClick={onClick} />)
+  fireEvent.click(getByText(defaultProps.text));
+  expect(onClick).toHaveBeenCalled();
+});
+
+// test('renders learn react link', () => {
```
Comment (abeer-f) [inline __tests__/CompTests/passwordEntryTest.jsx:61]: Do we need this comment?

File: __tests__/supertest.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    });
+  });
+
+  describe("/api/login", () => {
+    describe("POST", () => {
+      it("responds with 200 status and application/json content type", () => {
+        return request(server)
+          .post("/api/login")
+          .send({username: "Regis2", passwordUser: "tarotcards"})
+          // .expect("Content-Type", /json/)
```
Comment (abeer-f) [inline __tests__/supertest.js:96]: Comment

File: app-flow.drawio [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<mxfile host="65bd71144e">
```
Comment (abeer-f) [inline app-flow.drawio:1]: I don't think we need this file

**Review by abeer-f — APPROVED** [review]

**Pull request merged**